docs/site/LoopBack-types.md

```diff
@@ -130,6 +130,9 @@ value is provided.
  The type name is case-insensitive; so for example you can use either \"Number\" or \"number\".
 " %}
 
+{% include note.html content="The support for `GeoPoint` is not finished yet.
+Related GH issue: [Cannot start the application. Error: Unsupported type: geopoint](https://github.com/strongloop/loopback-next/issues/1981)" %}
```

Comment:
```suggestion
{% include note.html content="`GeoPoint` is not supported. See GitHub issue
[#1981](https://github.com/strongloop/loopback-next/issues/1981)" %}
```

Let's consider rephrasing it to be more concise.

Reply:
I was guessing that it's under development and will be available soon. If it is not supported at all, wouldn't it be better if we remove all the documentation about the GeoPoint type.

Reply:
Apologies for the misunderstanding.

The way I interpret it is: under development ≠ supported

For something to be supported, it needs to be developed. From the perspective of the user, the feature is not supported, regardless if it's under development or not.

The link to the open GitHub issue acts as a signal the feature is under consideration or development.

Hence the suggestion to just say "not supported" just to be concise.

Reply:
Alright, thank you for making it clearer. I am committing your suggestion.
Very minor change but my very first contribution to an oss ever! Hope I can contribute more! 👍